package.json

```
@@ -36,8 +36,7 @@
     "lint:fix": "eslint --max-warnings 0 --config .eslintrc . --fix",
     "generate:assets": "npm run generate:readme:toc",
     "generate:readme:toc": "markdown-toc -i README.md",
-    "generate:docker": "docker build --tag asyncapi/server-api .",
-    "release": "npm run generate:docker && semantic-release"
+    "docker:build": "docker build --tag asyncapi/server-api ."
```

```suggestion
    "docker:build": "docker build --tag asyncapi/server-api ."
    "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
```

please commit it as this is one more missing script that is required

---

@derberg Done